snippets/logging-outputs-5.6-snip.adoc

```diff
@@ -0,0 +1,28 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+.Output Destinations
+[options="header"]
```

Comment:
The coverage are different with QE. Could you confirm?

https://docs.google.com/spreadsheets/d/1uqtlotjNw1mJcIdOJrxuN6Sm9SQ9htH0htQ5QBauuUY/edit#gid=993623012

Reply:
Updated, please review.

Reply:
I think this is still not updated for 5.6. We have also tested Log 5.6 with kafka 3.3.1, Loki needs to be updated as well. 
Can you take a look at columns O - U on above 5.6 coverage doc.